src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -447,6 +448,19 @@ public Course getCourseViaExerciseGroupOrCourseMember() {
         }
     }
 
+    /**
+     * Utility method to get the exam. Get the exam over the exerciseGroup, if one was set, otherwise return null.
+     *
+     * @return exam, to which the exercise belongs
+     */
+    @JsonIgnore
+    public Exam getExamIdViaExerciseGroupOrCourseMember() {
```

Comment:
The method name sounds wrong. No `Id` is involved

Reply:
Yes, you are correct.